src/constants.ts

```diff
@@ -284,3 +284,8 @@ export const LAUNCHPEG_LENS_ADDRESS: { [chainId in ChainId]: string } = {
   [ChainId.FUJI]: '0x674ABC8D4aaCba9cE74E0304c831B10e7Fe4D646',
   [ChainId.AVALANCHE]: '0x7f281f22eDB332807A039073a7F34A4A215bE89e'
 }
+
+export const AUCTION_HOUSE: { [chainId in ChainId]: string } = {
+  [ChainId.FUJI]: '0x97D13c0ce96C1FAB9613cc654165B6474CbBEE59',
```

Comment:
the fuji one is actually `0x768Ba583BC928C6a691FAaD7E924c94E6E6F4F98`

Reply:
thanks!